app/frontend/src/features/communities/hooks.ts

```diff
@@ -160,6 +167,78 @@ export const useNewDiscussionMutation = (onSuccess?: () => void) => {
   );
 };
 
+export const useListAvailableReferences = (
```

Comment:
Don't we already have this hook somewhere? In profiles?

Reply:
No we didn't!